### PR TITLE
mcservman: Add version 0.0.6

### PR DESCRIPTION
**Pull request by CrazyWillBear**
Hey there! I think I fixed whatever error you picked up on, as I've just tested the .json and it worked. Thanks!

### CODE REVIEW

File: bucket/mcservman.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "MIT",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://github.com/CrazyWillBear/MinecraftServerManager/releases/download/v0.0.6-alpha/v0.0.6-alpha.zip#/mcservman.zip",
+    "hash": "ddd3bba7ca8861abccefed9778cfb9c6d850a753c699851c225c58ecbb52b58e",
+    "bin": [ [ "MCServMan.exe", "mcservman" ] ],
```
Comment (Ash258) [inline bucket/mcservman.json:14]: Why??

File: bucket/mcservman.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/mcservman.json:15]: Cannot work

File: bucket/mcservman.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "autoupdate": {
+        "url": "https://github.com/CrazyWillBear/MinecraftServerManager/releases/download/$version-portable/$version-portable.zip#/mcservman.zip"
```
Comment (Ash258) [inline bucket/mcservman.json:17]: rewrite zip to zip does not make sense

**Review by Ash258 — CHANGES_REQUESTED** [review]
Move this PR to https://github.com/Calinou/scoop-games

File: bucket/mcservman.json (hunk 1 above)
Comment (CrazyWillBear) [reply bucket/mcservman.json:14]: ?

File: bucket/mcservman.json (hunk 2 above)
Comment (CrazyWillBear) [reply bucket/mcservman.json:15]: What do you mean? It works for me when I test it

File: bucket/mcservman.json (hunk 3 above)
Comment (CrazyWillBear) [reply bucket/mcservman.json:17]: Where?

**Comment by CrazyWillBear** [comment]
@Ash258 this isn't a game, it's a server manager. Can I keep it in extras?

**Comment by Ash258** [comment]
> Move this PR to https://github.com/Calinou/scoop-games

**Comment by CrazyWillBear** [comment]
Okay

**Pull request closed without merging**